Dockerfile

```diff
@@ -13,7 +15,7 @@ RUN apt-get update && \
     apt-get update && \
     apt-get install -y adoptopenjdk-8-hotspot && \
```

Comment:
When I build and run the container locally, I get this output:

```
jenkins@d0f69f4b63fd:~$ java -version
openjdk version "11.0.14.1" 2022-02-08
OpenJDK Runtime Environment Temurin-11.0.14.1+1 (build 11.0.14.1+1)
OpenJDK 64-Bit Server VM Temurin-11.0.14.1+1 (build 11.0.14.1+1, mixed mode)
jenkins@d0f69f4b63fd:~$ javac -version
javac 11.0.14.1
jenkins@d0f69f4b63fd:~$ atlas-version

ATLAS Version:    8.2.8
ATLAS Home:       /usr/share/atlassian-plugin-sdk-8.2.8
ATLAS Scripts:    /usr/share/atlassian-plugin-sdk-8.2.8/bin
ATLAS Maven Home: /usr/share/atlassian-plugin-sdk-8.2.8/apache-maven-3.5.4
AMPS Version:     8.1.2
--------
Executing: /usr/share/atlassian-plugin-sdk-8.2.8/apache-maven-3.5.4/bin/mvn --version -gs /usr/share/atlassian-plugin-sdk-8.2.8/apache-maven-3.5.4/conf/settings.xml
Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z)
Maven home: /usr/share/atlassian-plugin-sdk-8.2.8/apache-maven-3.5.4
Java version: 11.0.14.1, vendor: Eclipse Adoptium, runtime: /opt/java/openjdk
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.10.104-linuxkit", arch: "aarch64", family: "unix"
```

This makes me think the AdoptOpenJDK8 package isn't being used. Could we remove this line, lines 12–14, and `adoptopenjdk.pub` from line 23?

Reply:
Good catch!